config-templates/template_list.json

```diff
@@ -30,5 +30,21 @@
     "default-version": "4.5",
     "enabled": "true",
     "provider": "redhat"
+  },
+  {
+    "name": "ess",
+    "displayname": "IBM Spectrum Scale Driver",
+    "Description": "IBM Spectrum Scale Driver to enable ESS storage.",
+    "default-version": "2.1.0",
+    "enabled": "true",
+    "provider": "ibm"
+  },
+  {
+    "name": "ibm-system-storage-block-csi-driver",
+    "displayname": "IBM block storage CSI driver",
+    "Description": "IBM block storage CSI driver for IBM Spectrum Virtualize, IBM FlashSystem and IBM DS8000 Family.",
+    "default-version": "1.4.0",
+    "enabled": "true",
+    "provider": "ibm"
```

Comment:
I think this is part of @ArbelNathan  PR ? https://github.com/IBM/ibm-satellite-storage/pull/105